torch/onnx/verification.py

```diff
@@ -1795,6 +1795,7 @@ def verify_model_with_fx_to_onnx_exporter(
     rtol: float = 0.001,
     atol: float = 1e-7,
     acceptable_error_percentage: Optional[float] = None,
+    opset_version: Optional[int] = None,
```

Comment:
I suppose you'd need to assign a default value if None?

Reply:
I notice the one under verify function is also None.